tox.ini

```diff
@@ -1,4 +1,4 @@
 [flake8]
-ignore = F401,E501,E402,E401
+ignore = F401,E501,E402,E401, E305, E302, N812
```

Comment:
I'm pretty sure the errors are:

    E302 expected 2 blank lines, found 0
    E305 expected 2 blank lines after class or function definition, found 1

Well, p00p!  I'm flaking out?

But from the moment you introduced me to flake8, I run it as part of my whole tests thing every single time.  So why wasn't it finding that missing line??

Under no circumstances allow me to slack off on flake8 styles.  It's very little effort to do it right.